src/services/mongoose/schema.ts

```diff
@@ -363,6 +363,8 @@ const paymentFlowStateSchema = new Schema<PaymentFlowStateRecord>(
     usdPaymentAmount: { type: Number, required: true },
     inputAmount: { type: Number, required: true },
 
+    walletPriceRatio: { type: String, required: true },
```

Comment:
do we need a migration?

Reply:
Maybe not because they only live for 5 mins in the collection and get deleted after. There will be an intermediate state though where if someone has done a fee query and then we deploy and then they try to pay that they'll see an error in the 5 mins after their fee query. 

This might be an edge case that may not be worth explicitly accommodating for?